### PR TITLE
Updates recovery dialog text and buttons

### LINKED ISSUE #11857: Ledger Wallet Recovery dialog needs updating
The text and button arrangement should be updated to reflect that attached mockup:

image

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves #11857 

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Test Plan:
- start browser
- enable payments
- go to recovery dialog
- make sure that it's the same as spec

**New look**
image

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by arsalankhalid** [comment]
Oh no, that sucks I did this :( I guess I took too long to get it covered as it was on my plate. I'll give your branch a whirl, although not sure why you removed all this stuff in preferences, with hiding overlays? 

if (overlayName === 'addFunds' && isVisible === false) {	
-      // Hide the child overlays when the parent is closed	
-      stateDiff['bitcoinOverlayVisible'] = false	
-      stateDiff['qrcodeOverlayVisible'] = false	
-    }

And also some first, second recovery keys.

**Comment by NejcZdovc** [comment]
@arsalankhalid this is just some dead old code that was there and I removed it.

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 1]
```diff
@@ -166,14 +164,14 @@ ledgerBackupText1=Below, you will find the anonymized recovery key that is requi
 ledgerBackupText2=Make sure you keep this key private, or else your wallet will be compromised.
 ledgerBackupTitle=Backup your Brave wallet
 ledgerPaymentsShown=Brave Payments
-ledgerRecoveryContent=Your previous wallet will now be used. Your new wallet will be discarded.
+ledgerRecoveryContent=Note: The recovered BAT wallet will be replace the current BAT wallet, which will be discarded.
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply app/extensions/brave/locales/en-US/preferences.properties]: fixed

**Comment by bradleyrichter** [comment]
The font on the footnote can be Regular and not Bold. Otherwise, it looks good!

**Comment by NejcZdovc** [comment]
@bradleyrichter remove bold, updated image in the first comment

**Comment by NejcZdovc** [comment]
master 36d892d
0.23 0d6419754
0.22 5564d3a37

**Pull request merged**